app/src/main/java/org/wikipedia/page/PageFragment.java

```diff
@@ -1001,8 +1001,10 @@ private void setupMessageHandlers() {
             }
         });
         bridge.addListener("scroll_to_anchor", (String messageType, JsonObject payload) -> {
-            int diffY = DimenUtil.roundedDpToPx(payload.getAsJsonObject("rect").get("y").getAsFloat());
-            webView.setScrollY(webView.getScrollY() + diffY - webView.getHeight() / 2);
+            if (payload.get("anchor").getAsString().contains("cite_")) {
```

Comment:
Why are we now restricting this for only references? Why not allow this handler to scroll to sections, as well?